mixlib-install.gemspec

```diff
@@ -14,12 +14,13 @@ Gem::Specification.new do |spec|
   spec.homepage      = "https://chef.io"
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.executables   = ["mixlib-install"]
   spec.require_paths = ["lib"]
 
   spec.add_dependency "artifactory"
   spec.add_dependency "mixlib-versioning"
   spec.add_dependency "mixlib-shellout"
+  spec.add_dependency "thor"
```

Comment:
`thor` is probably a better choice as `test-kitchen` already uses it (and has a dep on `mixlib-install`). This will ensure we don't add an additional `mixlib-cli` (transitive) dep to `test-kitchen`.